battleground.py

```diff
@@ -45,38 +45,32 @@ def __init__(self, threadID, name, server, bot_1, bot_2):
     def saveData(self, bot_1, bot_2):
         self.fileLogName = f'{self.threadID}.json'
         Path("./history/games/").mkdir(parents=True, exist_ok=True)
-        game_json = tree()
-        game_json['ROUNDS'] = []
-        for round in self.gameRecord:
-            game_json['ROUNDS'].append(round)
-        with open('./history/games/' + self.fileLogName, 'w') as file:
-            file.writelines(json.dumps(game_json, sort_keys=True, indent=4))
+        with open(f'./history/games/{self.fileLogName}', 'w') as file:
+            file.writelines(json.dumps(self.gameRecord, sort_keys=True, indent=4))
 
         try:
             with open('./history/game_list.json', 'r') as file:
                 game_list = ''
                 for line in file:
                     game_list += line
                 game_list = json.loads(game_list)
-        except FileNotFoundError:
-            logging.info('No game_list.json file. Creating file.')
-            game_list = tree()
-            game_list['GAMES'] = []
-        except json.JSONDecodeError:
-            logging.info('Bad format inside game_list.json file.')
-            game_list = tree()
-            game_list['GAMES'] = []
+        except (FileNotFoundError, json.JSONDecodeError) as error:
+            if isinstance(error, FileNotFoundError):
+                logging.info('No game_list.json file. Creating file.')
+            else:
+                logging.info('Bad format inside game_list.json file.')
+            game_list = []
 
         with open('./history/game_list.json', 'w') as file:
             data = tree()
-            data['GAME_ID'] = self.threadID
-            data['BOT_1']['NAME'] = bot_1.name()
-            data['BOT_1']['POINTS'] = bot_1.points()
-            data['BOT_2']['NAME'] = bot_2.name()
-            data['BOT_2']['POINTS'] = bot_2.points()
-            data['ROUNDS'] = self.passedRounds
-            data['DATE'] = datetime.now().strftime('%Y-%m-%dT%H:%M:%S')
-            game_list['GAMES'].append(data)
+            data[GamesListFileField.GAME_ID.value] = self.threadID
+            data[GamesListFileField.BOT_1.value][GamesListFileField.BOT_NAME.value] = bot_1.name()
```

Comment:
These lines are too long. Try to keep up to 80 chars in single line of source, no more.
On the other hand: it's just selecting an element from a two-dimensional array, and it already takes a whole line of 80 characters - something is really off with it...